### PR TITLE
Better error handling / filePath operations

### LINKED ISSUE #18: Confusing error message when vm has more than one user-assigned identities
When virtual machine has two on more user-assigned managed identities then azmitool setblob fails with confusing error message: "General Error: The authentication request failed due to an unhandled exception.  See inner exception for details."

Exception catching in azmi-commandline.cs should not be so generic and provide more meaningful information by exception.

### LINKED ISSUE #52: Use file operations in get/set blob
Instead of stream, use directly file path argument.

### PR DESCRIPTION
**Pull request by iricigor**
Following features implemented
- getToken error handling, resolves #18 
- getBlob / setBlob uses filePath operations, better error handling, also resolves #52 

Code changes:
- new method `IdentityError` to handle identity related error messages
- simplified setBlob / getBlob commands to use filePath method operations
- removed sections which just re-throw the same error

### CODE REVIEW

**Comment by iricigor** [comment]
[Build ](https://skype.visualstudio.com/SCC/_build/results?buildId=26950205) and [Integration tests ](https://skype.visualstudio.com/SCC/_build/results?buildId=26950494&view=ms.vss-test-web.build-test-results-tab) successful.

No new tests added, as this introduces no new features. It modifies error messages and simplifies code.

**Comment by iricigor** [comment]
In action
image

File: README.md [hunk 1]
```diff
@@ -51,6 +51,23 @@ Azmi is not working across different AAD tenants.
 Read more:
 - [Managed identities for Azure resources](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
  
+## Common errors
+
+By default, `azmi` will display simple, Linux style errors. To discard the, you can redirect error stream.
```
Comment (MangyCoyote) [inline README.md]: them?

Comment (iricigor) [reply README.md]: Fixed

**Pull request merged**